pkgs/development/python-modules/optuna-dashboard/default.nix

```diff
@@ -58,6 +58,12 @@ buildPythonPackage rec {
     streamlit
   ];
 
+  disabledTests = lib.optionals stdenv.hostPlatform.isDarwin [
+    # AttributeError: module 'numpy' has no attribute 'float128' ==> not available on 64-bit Darwin
```

Comment:
Out of curiosity: are there non 64-bit darwin in the wild?

Reply:
Ah, I mentioned "64-bit" to make it explicit that 64 != 128